cmd/fabric8-jenkins-proxy/main_cors_test.go

```diff
@@ -31,37 +27,54 @@ func TestAPIServerCORSHeaders(t *testing.T) {
 
 	reader, _ := http.NewRequest("POST", "/doesntmatter", nil)
 
-	// Check for origin "https://*.openshift.io"
+	// hope UUID generates valid characters for domain name
```

Comment:
why are we hoping instead of asserting ?

Reply:
Absolutely. _ignored somehow_